middle_end/flambda2/types/grammar/more_type_creators.ml

```diff
@@ -148,8 +146,9 @@ let immutable_block_with_size_at_least ~tag ~n ~field_kind ~field_n_minus_one =
   TG.create_variant ~is_unique:false
     ~immediates:(Known (bottom K.naked_immediate))
     ~blocks:
-      (Known (TG.Row_like_for_blocks.create ~field_kind ~field_tys (Open tag)))
-    Unknown
+      (Known
+         (TG.Row_like_for_blocks.create ~field_kind ~field_tys (Open tag)
+            Unknown))
```

Comment:
Is there a reason this doesn't take `alloc_mode` (like the other functions) instead of using `Unknown`?

Reply:
Yes. This is the function used to create shapes for the `Block_load` primitive, which doesn't carry any allocation mode. We could change the interface to force the callers to explicitly pass an `Unknown` alloc mode, of course, but in the context of this PR it made sense to keep the original interface.